package.json

```diff
@@ -16,10 +16,12 @@
     "node": ">=17"
   },
   "dependencies": {
+    "@yext/yext-sites-scripts": "github:mkilpatrick/yext-sites-scripts#aliang-multilang",
```

Comment:
Don't actually commit pointing to your PR since once it's merged this won't be what we want